pom.xml

```diff
@@ -50,6 +50,22 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>net.revelc.code.formatter</groupId>
+                <artifactId>formatter-maven-plugin</artifactId>
+                <version>${formatter.maven.plugin.version}</version>
+                <executions>
```

Comment:
Early, I had issue with validation code using that formatter, and we agreed with mentors to remove this validation step.
But I see, that we need formatter in the docker file.
So, could you add a plugin but don't call it in the validation stage:
```
......
<executions>
        <execution>
            <id>format</id>
            <phase>none</phase>
        </execution>
    </executions>
.....
```